code/code/sys/comm.cc

```diff
@@ -258,6 +259,52 @@ void TBeing::sendMaxStatsGmcp() const {
   desc->sendGmcp(maxStats, false);
 }
 
+void TBeing::sendStatusGmcp() const {
+  if (!desc)
+    return;
+
+  float xp = static_cast<float>(getExp());
+  float xptnl =
+    static_cast<float>(getExpClassLevel(GetMaxLevel() + 1) - getExp());
+  float levelxp = static_cast<float>(getExpClassLevel(GetMaxLevel()));
+  float nextlevelxp = static_cast<float>(getExpClassLevel(GetMaxLevel() + 1));
+
+  sstring charData = format(
+                       "char.status {"
```

Comment:
There would be some slight benefit in unifying this format with other MUDs. Here's an example: https://www.aardwolf.com/wiki/index.php/Clients/GMCP#aard_char_worth

IRE muds also have public GMCP docs, I hear (though I could only find https://nexus.ironrealms.com/GMCP#Char which doesn't really describe it) which we could emulate.

Reply:
_Interesting!_ Yeah, I can restructure things to align more closely with those examples, at least for some of the data.